client/src/components/dashboard/OverviewPage.jsx

```diff
@@ -38,4 +54,8 @@ const mapStateToProps = (state) => ({
   hasTriedLogin: user.hasTriedLogin(state),
 });
 
-export default withStyles(styles)(connect(mapStateToProps)(OverviewPage));
+const mapDispatchToProps = (dispatch) => ({
```

Comment:
@martinjoha We don't need to use this, just do 
`const { actions } = this.props;
actions.sendEmail(subject, body)`
whenever needed